### ISSUE TITLE
Documentation shows "keras.layers.0)" instead of actual class header in several cases.

### ISSUE THREAD

**Issue by tjochmann** [keras-team/keras#8266]
In some cases, the auto-generated class headers are wrong. For example, the Masking class shows _keras.layers.0)_ instead.

Examples:

- https://keras.io/layers/core/#masking
- https://keras.io/layers/core/#activityregularization

**Comment by kgruhler** [comment]
Same at Recurrent Layers Page: [https://keras.io/layers/recurrent/](https://keras.io/layers/recurrent/)

**Comment by charlesreid1** [comment]
See PR fchollet/keras/pull/8265. This problem has been fixed on the master branch. The PR linked to should create a unit test to fix this in the future.
